compiler/src/dmd/backend/aarray.d

```diff
@@ -22,7 +22,7 @@ alias hash_t = size_t;
 version (MARS)
 {
     import dmd.root.hash;
-    import dmd.backend.global : err_nomem;
+    import dmd.backend.ph2 : err_nomem;
 }
```

Comment:
Perhaps `err_nomem` should be moved to `dmd.backend.global`? It feels like a more central utility function.

Reply:
I'm now importing them through `dmd.backend.global` again using the public import